CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## v0.64.0 - 2022-04-29
+
+- Fix bug in merge - merge with no changes resolves by creating empty commit (with no data) on destinations head  (#3270)
+- Fix broken content-based diff for changed objects in compare view (#3275)
+- Bump metadata client version to 0.1.7-RC.0 (#3277)
```

Comment:
GC isn't released together with a lakeFS release, right? In that case I think we should avoid adding it to the lakeFS changelog (But there's really no need to fix it now).

Reply:
Yup, my mistake there.  OTOH it _does_ sort-of state that we are now testing after cutting this release with lakeFS (we always test HEAD, but obviously a release is a nice step to indicate along the way...).